packages/gatsby-plugin-offline/package.json

```diff
@@ -34,8 +35,10 @@
   },
   "repository": "https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-offline",
   "scripts": {
-    "build": "babel src --out-dir . --ignore **/__tests__",
+    "build": "npm run build:src && npm run build:sw-append",
+    "build:src": "babel src --out-dir . --ignore **/__tests__",
```

Comment:
do we need to ignore `sw-append.js` here? for builds it probably doesn't matter - it will be overwritten by cpx in next step, but while watching, babel will try to compile it?

Reply:
ah yep, that's a good point - need to fix that